### ISSUE TITLE
EventBanner flyers should have better alt text

### ISSUE THREAD

**Issue by dyc3** [StevensSEC/sec.club#79]
The `EventBanner` component should have an optional prop to set the alt text of the flyer. If this prop is not specified, the alt text should be set to something other than the text
